家計簿アプリのビルドエラー直して

@Claude
家計簿アプリビルドエラーだから直して
エラーは以下の通り

Failed to compile.

./app/household/page.tsx:345:52
Type error: Property 'category' does not exist on type 'PieLabelRenderProps'.

  343 |                         cy="50%"
  344 |                         outerRadius={100}
> 345 |                         label={(entry) => `${entry.category}: ${formatCurrency(entry.amount)}`}
      |                                                    ^
  346 |                         labelLine={false}
  347 |                       >
  348 |                         {expenseByCategory.map((entry, index) => (
Next.js build worker exited with code: 1 and signal: null
Error: Process completed with exit code 1.

--- a/app/household/page.tsx
+++ b/app/household/page.tsx
@@ -342,7 +342,7 @@ export default function HouseholdPage() {
                         cx="50%"
                         cy="50%"
                         outerRadius={100}
-                        label={(entry) => `${entry.category}: ${formatCurrency(entry.amount)}`}
+                        label={(entry: { category: string; amount: number }) => `${entry.category}: ${formatCurrency(entry.amount)}`}
                         labelLine={false}
                       >
                         {expenseByCategory.map((entry, index) => (
@@ -370,7 +370,7 @@ export default function HouseholdPage() {
                         cx="50%"
                         cy="50%"
                         outerRadius={100}
-                        label={(entry) => `${entry.category}: ${formatCurrency(entry.amount)}`}
+                        label={(entry: { category: string; amount: number }) => `${entry.category}: ${formatCurrency(entry.amount)}`}
                         labelLine={false}
                       >
                         {incomeByCategory.map((entry, index) => (

